Input/Qualification-Ontogeny-UGT1A9.json

```diff
@@ -144,381 +139,122 @@
-                                "Output": "Organism|PeripheralVenousBlood|Propofol|Plasma (Peripheral Venous Blood)",
-                                "ObservedData": "PK-Parameter",
-                                "ObservedDataRecordId": 5366
-                            },
-                            {
-                                "Project": "Propofol-Pediatrics",
-                                "Simulation": "Saint-Maurice 1989 ind.6",
-                                "Output": "Organism|PeripheralVenousBlood|Propofol|Plasma (Peripheral Venous Blood)",
-                                "ObservedData": "PK-Parameter",
-                                "ObservedDataRecordId": 5367
-                            },
-                            {
-                                "Project": "Propofol-Pediatrics",
-                                "Simulation": "Saint-Maurice 1989 ind.7",
-                                "Output": "Organism|PeripheralVenousBlood|Propofol|Plasma (Peripheral Venous Blood)",
-                                "ObservedData": "PK-Parameter",
-                                "ObservedDataRecordId": 5368
-                            },
-                            {
-                                "Project": "Propofol-Pediatrics",
-                                "Simulation": "Saint-Maurice 1989 ind.8",
-                                "Output": "Organism|PeripheralVenousBlood|Propofol|Plasma (Peripheral Venous Blood)",
-                                "ObservedData": "PK-Parameter",
-                                "ObservedDataRecordId": 5369
-                            },
-                            {
-                                "Project": "Propofol-Pediatrics",
-                                "Simulation": "Saint-Maurice 1989 ind.9",
-                                "Output": "Organism|PeripheralVenousBlood|Propofol|Plasma (Peripheral Venous Blood)",
-                                "ObservedData": "PK-Parameter",
-                                "ObservedDataRecordId": 5370
-                            },
-                            {
-                                "Project": "Propofol-Pediatrics",
-                                "Simulation": "Saint-Maurice 1989 ind.10",
-                                "Output": "Organism|PeripheralVenousBlood|Propofol|Plasma (Peripheral Venous Blood)",
-                                "ObservedData": "PK-Parameter",
-                                "ObservedDataRecordId": 5434
-                            }
-                        ]
-                    }
-                ]
-            },
             {
                 "Title": "Overall predictivity of the propofol PBPK model. Open circles represent mean ratios of PBPK predicted AUC over observed AUC of propofol in children 8 days to 10 years old. Blue dashed lines and red dotted lines represent the 1.5-fold and 2-fold error, respectively.",
                 "PKParameter": "AUC",
```

Comment:
This is still accepted, but to make it conform with the new schema, it should be
`"PKParameters": ["AUC"],`